tracer/src/Datadog.Trace/ClrProfiler/AutoInstrumentation/GraphQL/GraphQLSecurityCommon.cs

```diff
@@ -0,0 +1,117 @@
+// <copyright file="GraphQLSecurityCommon.cs" company="Datadog">
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
+// </copyright>
+
+using System;
+using System.Collections.Concurrent;
+using System.Collections.Generic;
+using System.Web;
+using Datadog.Trace.AppSec;
+using Datadog.Trace.AppSec.Coordinator;
+
+namespace Datadog.Trace.ClrProfiler.AutoInstrumentation.GraphQL;
+
```

Comment:
NIT: #nullable enable can help to prevent null reference exceptions

Reply:
agreed, we should add `#nullable enable` to all new files in Datadog.Trace

Reply:
All graphql files now use `#nullable enable`, see 8e67bc2d3bf40bedb468c49a28e474862c23c105